test/runnable/circular.d

```diff
@@ -1,7 +1,14 @@
+
 // REQUIRED_ARGS: -d
 // PERMUTE_ARGS: -dw
 // EXTRA_SOURCES: imports/circularA.d
 // This bug is typedef-specific.
```

Comment:
Ahem maybe this test is useless now...

Reply:
Probably. Not sure why it's a runnable test anyway because it targets a compiler error

Reply:
But  pruning the test suite is something for another PR IMHO.